### DISCUSSION TITLE
Black hole unit

### DISCUSSION THREAD

**Discussion by Royalexo** [InnovativeOnlineIndustries/Industrial-Foregoing#1178] [Completed/Done]
add the lock item feature to black holes that only that 1 item/fluid goes in

**Comment by Buuz135** [comment]
Have you tried clicking the blue slot?

**Comment by Royalexo** [comment]
Yes i have tried that with an oak and dark aok sapling 
i am using refined pipes mod to send the items to the black hole unit

**Comment by Royalexo** [comment]
i also hold right mousebutton  while a item was in the blue slot and another item went in it
